src/lints/enum_tuple_variant_field_marked_deprecated.ron

```diff
@@ -79,4 +83,10 @@ SemverQuery(
     },
     error_message: "A field in an enum's tuple variant is now #[deprecated]. Downstream crates will get a compiler warning when accessing this field.",
     per_result_error_template: Some("field {{enum_name}}::{{variant_name}}.{{field_name}} in {{span_filename}}:{{span_begin_line}}"),
+    witness: (
+        hint_template: r#"match value {
+    {{ join "::" path }}::{{ variant_name }} ({{#each baseline_field_names }}{{#if (eq this ../field_name)}}to_be_deprecated{{else}}_{{/if}}{{#unless @last }}, {{/unless}}{{/each}}) => (),
+    _ => (),
+}"#,
```

Comment:
Excellent use of templating, well done!

Adding my suggestion to name the deprecated field `witness`:
```suggestion
        hint_template: r#"match value {
    {{ join "::" path }}::{{ variant_name }} ({{#each baseline_field_names }}{{#if (eq this ../field_name)}}witness{{else}}_{{/if}}{{#unless @last }}, {{/unless}}{{/each}}) => (),
    _ => (),
}"#,
```